vscode/src/gh-copilot/instructions.ts

```diff
@@ -15,6 +15,48 @@ const codingInstructionsContent = `${codingInstructionsTitle}
 Follow these instructions when generating Q# code in .qs files,
 and Q# project folders that include a \`qsharp.json\` file.
 
+## Syntax
+
+When writing Q#, the \`for (i in 1..100)\` syntax is invalid, use \`for i in 1..100\` or
+\`for element in array[2...]\`.
+
+While many Q# operators are C-like, it uses \`or\` instead of \`||\` and \`and\` instead of \`&&\`.
+
+To extract values from a tuple, use destructuring via the \`let (item0, item1) = tupleValue;\` syntax.
+
+## Project structure
+
+### Single-file projects
+
+Q# files don't always need to exist in a project. A single \`.qs\` file can be compiled and
+run without a \`qsharp.json\` file. Prefer a single \`.qs\` file for simple programs.
+
+### Multi-file projects
+
+When Q# source files need to to reference each other, a \`qsharp.json\` file must be
+created. Source files must exist under the \`src\` folder.
+
+Example layout:
+
+\`\`\`
+project_root
+|--qsharp.json
+|--src
+|--|--Main.qs
+|--|--Tests.qs
+\`\`\`
+
+A typical \`qsharp.json\` will be an empty JSON file.
```

Comment:
```suggestion
A typical \`qsharp.json\` will be a JSON file with an empty JSON object in it.
```
I worry that it will get confused and think to make a file that is actually empty, rather than a file that contains `{}`. I see the example below clarifies this, but it'd be good to be as consistent and clear as can be.

Reply:
I like that, thanks.